presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -876,4 +897,18 @@ Hive Connector Limitations
 
 * :doc:`/sql/delete` is only supported if the ``WHERE`` clause matches entire partitions.
 * :doc:`/sql/alter-schema` usage fails, since the Hive metastore does not support renaming schemas.
+* Writing to and creation of transactional tables is not supported.
```

Comment:
It seems better to move this to the *Supported Table Types* section, as it's most relevant there. The limitations section here is more about SQL related features that are not supported.